recipe/meta.yaml

```diff
@@ -20,10 +20,10 @@ requirements:
     - make
   host:
     - zlib
-    - openssl
+    - openssl {{ openssl }}
   run:
     - zlib
-    - openssl
```

Comment:
personally I would prefer using pin_compatible. The `{{ openssl }}` in the `host` section is un-explicit as to the version number anyway

Reply:
Yeah, it's a weird balance to please people that like aggregate and please people that like explicitness. Also, I'm not sure but I think `pin_compatible` isn't smart enough to use what's defined in the run_exports... Or at least I think that's what I observed.